pretty/src/pane.rs

```diff
@@ -19,9 +19,13 @@ pub enum PaneSize {
 }
 
 #[derive(Clone)]
+#[non_exhaustive]
 pub enum Content {
```

Comment:
why non_exhaustive?

Reply:
So that adding new variants won't be a breaking change. When I added these most recent variants, I had to go edit some tests' `get_contents` closures that tried to match on all variants, even though they only had documents for 1 or 2 of the variants. Now they'll be forced to include a `_ => None` case, which is the right thing to do.